packages/stryker-html-reporter/package.json

```diff
@@ -49,10 +49,12 @@
   },
   "devDependencies": {
     "@types/file-url": "^1.0.28",
+    "@types/jsdom": "^11.0.2",
```

Comment:
What do we do with types that are only used in one package? Do we add them to the package or to the root package.json for lerna?

Reply:
These types cannot be added to the root package.json, as it would force `lib: ['dom']` for all packages. 

The most important thing is to keep the types where we also keep de (dev) dependencies. For example: i had to add `'@types/jsdom'` here, so i also added `'jsdom'` in this same package.